### PR TITLE
XS-4059 Removing three elements for DQC 0015

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Removed three elements after public comment period.
- Change_two with reasoning ...

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [x] If this is a code change, reviewed by a second person. (+1)
- [x] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: dqc_us_rules/resources/DQC_US_0015/dqc_15_concepts.csv [hunk 1]
```diff
@@ -6040,4 +6037,4 @@
 6325,SharesSubjectToMandatoryRedemptionSettlementTermsExcessOfLiabilitiesOverAssets
 6326,DeferredPurchasedPowerCosts
 6327,LossRecognizedOnAssetsTransferredToSeparateAccountGross
-6328,NonallowableAssets
\ No newline at end of file
+6328,NonallowableAssets
```
Comment (brianmyers-wf) [inline dqc_us_rules/resources/DQC_US_0015/dqc_15_concepts.csv]: yeah, that's a Window's new line, should probably remove/replace it

**Comment by brianmyers-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by joyreistad-wf** [comment]
+1

**Comment by derekgengenbacher-wf** [comment]
+1

**Comment by sagesmith-wf** [comment]
#### Testing Notes:
- [ ] Travis Passed
- [ ] Integration Tests Passed
- [ ] Local run in Arelle fired only one error on the filing it previously fired 4

**Comment by sagesmith-wf** [comment]
QA +1

**Comment by jasonleinbach-wf** [comment]
All checks pass.  Merging.

**Pull request merged**